### PR TITLE
Math Magicians: refactor with hooks

### PR DESCRIPTION
**Pull request by denisdiaconu**
In this project, I refactored the Calculator component that I developed in the past projects using hooks.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,86 +1,77 @@
...
-          </div>
-          <div className="conFifth">
-            <button className="colorW" type="button" name="0" onClick={(e) => { this.getInf(this.state, e.target.name); }}>0</button>
-            <div>
-              <button className="colorW" type="button" name="." onClick={(e) => { this.getInf(this.state, e.target.name); }}>.</button>
-              <button className="colorS" type="button" name="=" onClick={(e) => { this.getInf(this.state, e.target.name); }}>=</button>
-            </div>
+  return (
+    <div className="calContainer">
+      <input className="calInput" placeholder="0" value={`${format()}`} disabled="disabled" />
```
Comment (vichuge) [inline src/components/Calculator.js:39]: - [x] It looks like there's a bug in the display calculator. When I try an operation like `5+6` the answer includes the sign for this at the end. I kindly invite you to work on this. 🐶 
image

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your PR messages are descriptive ✔️

## Aspects to improve ♻️

- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by denisdiaconu** [comment]
Hello!
I explained in the last milestone too that this is not a bug.
The response comes with an operator so the user can do more calculations.

**Comment by vichuge** [comment]
> Hello!
> I explained in the last milestone too that this is not a bug.
> The response comes with an operator so the user can do more calculations.

I understand, according to the requirements for this project one point is `Run the app, is everything running smoothly as it should?`. I just try with 2 different calculators and any of them save the operator to the next one. I kindly invite you to change it for the next revision.

File: src/components/Calculator.js (hunk 1 above)
Comment (vichuge) [inline src/components/Calculator.js:39]: - [ ] Please, work on this change and you'll be ready for the next project. I answer your comment in this same PR 😸

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ♻️ ⚠️ ⚠️ 🚦 🚦 

Hi dear student, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

star_wars_almost_there


## Aspects to improve ♻️
_Check the comments under the review._

- [ ] Check the comments under the review. 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**